utils/hct/hctdb_instrhelp.py

```diff
@@ -180,11 +180,11 @@ class db_instrhelp_gen:
     def __init__(self, db):
         self.db = db
         self.llvm_type_map = {
-            "i1": "bool",
-            "i8": "int8_t",
-            "u8": "uint8_t",
-            "i32": "int32_t",
-            "u32": "uint32_t"
+            "i1": ("bool", 1),
```

Comment:
Would be nice to use a [namedtuple](https://docs.python.org/2/library/collections.html#collections.namedtuple) or class here so we can access them by name instead of by position.